src/sage/combinat/designs/incidence_structures.py

```diff
@@ -1924,8 +1924,8 @@ def is_resolvable(self, certificate=False, solver=None, verbose=0, check=True,
             sage: TD.is_resolvable()
```

Comment:
Remaining doctest warnings in this file:
```
sage -t --long --warn-long 33.8 --random-seed=61144299382637325659043318932957598596 src/sage/combinat/designs/incidence_structures.py
**********************************************************************
File "src/sage/combinat/designs/incidence_structures.py", line 1541, in sage.combinat.designs.incidence_structures.IncidenceStructure.is_t_design
Warning: Consider using a block-scoped tag by inserting the line 'sage: # optional - gap_package_design' just before this line to avoid repeating the tag 5 times
    BD = designs.WittDesign(9)         # optional - gap_package_design
    [344 tests, 2.08 s]
```

Reply:
Fixed in 67eb2e1f888